homework/03LessonThree.js

```diff
@@ -110,10 +111,14 @@ var threeByThreeArray = [
 
 // Problem 11:
 // Replace the center element of threeByThreeArray, which is 5, with something else.
+threeByThreeArray[1,1] = '8';
```

Comment:
using comma-separated values inside your bracket notations will catch up with you. stick to just using one value in bracket notation: `threeByThreeArray[2] = '8'`